concepticondata/conceptlists/Morse-1996-375.tsv

```diff
@@ -0,0 +1,376 @@
+Morse-1996-375-284	284	volar	fly		1441	FLY (MOVE THROUGH AIR)
+Morse-1996-375-285	285	andar	walk		1443	WALK
+Morse-1996-375-286	286	venir	come		1446	COME
+Morse-1996-375-287	287	acostado	lie		601	LIE (MISLEAD)
+Morse-1996-375-288	288	sentarse, sentado	sit		1416	SIT
+Morse-1996-375-289	289	ponerse en pie, de pie	stand		570	STAND UP
+Morse-1996-375-290	290	dar	give		1447	GIVE
+Morse-1996-375-291	291	decir	say		1458	SAY
+Morse-1996-375-292	292	arder	burn		2102	BURN
+Morse-1996-375-293	293	montaña	mountain		639	MOUNTAIN
+Morse-1996-375-294	294	rojo	red		156	RED
+Morse-1996-375-295	295	verde	green		1425	GREEN
+Morse-1996-375-296	296	amarillo	yellow		1424	YELLOW
+Morse-1996-375-297	297	lleno	full		1429	FULL
+Morse-1996-375-298	298	nuevo	new		1231	NEW
+Morse-1996-375-299	299	redondo	round		1395	ROUND
+Morse-1996-375-300	300	seco	dry		1398	DRY
+Morse-1996-375-301	301	nombre	name		1405	NAME
+Morse-1996-375-302	302	como	how		1239	HOW
+Morse-1996-375-303	303	cuando	when		1238	WHEN
+Morse-1996-375-304	304	donde	where		1237	WHERE
+Morse-1996-375-305	305	aquí	here		136	HERE
+Morse-1996-375-306	306	allí	there		1937	THERE
+Morse-1996-375-307	307	otro	other		197	OTHER
+Morse-1996-375-308	308	pocos	few		1242	FEW
+Morse-1996-375-309	309	neblina	fog		249	FOG
+Morse-1996-375-310	310	fluir	flow		2003	FLOW
+Morse-1996-375-311	311	mar	sea		1474	SEA
+Morse-1996-375-312	312	mojado	wet		1726	WET
+Morse-1996-375-313	313	lavar	wash		1453	WASH
+Morse-1996-375-314	314	gusano	worm		1219	WORM
+Morse-1996-375-315	315	ala	wing		1257	WING
+Morse-1996-375-316	316	piel peluda	fur		580	FUR
+Morse-1996-375-317	317	ombligo	navel		1838	NAVEL
+Morse-1996-375-318	318	saliva	saliva		1503	SALIVA
+Morse-1996-375-319	319	leche	milk		635	MILK
+Morse-1996-375-320	320	con	with		1340	WITH
+Morse-1996-375-321	321	en	in		1460	IN
+Morse-1996-375-322	322	en, lugar definido	at		1461	AT
+Morse-1996-375-323	323	si	if		1459	IF
+Morse-1996-375-324	324	hielo	ice		617	ICE
+Morse-1996-375-325	325	nieve	snow		784	SNOW
+Morse-1996-375-326	326	helarse	freeze		1431	FREEZE
+Morse-1996-375-327	327	niño	child		2099	CHILD
+Morse-1996-375-328	328	oscuro	dark		706	DARK
+Morse-1996-375-329	329	cortar	cut		1432	CUT
+Morse-1996-375-330	330	ancho	wide		1243	WIDE
+Morse-1996-375-331	331	estrecho	narrow		1267	NARROW
+Morse-1996-375-332	332	lejos	far		1406	FAR
+Morse-1996-375-333	333	cerca	near		1942	NEAR
+Morse-1996-375-334	334	grueso	thick		1244	THICK
+Morse-1996-375-335	335	delgado	thin		1400	THIN (SLIM)
+Morse-1996-375-336	336	corto	short		1645	SHORT
+Morse-1996-375-337	337	pesado	heavy		1210	HEAVY
+Morse-1996-375-338	338	embotado	dull		379	BLUNT
+Morse-1996-375-339	339	afilado	sharp		1396	SHARP
+Morse-1996-375-340	340	sucio	dirty		1230	DIRTY
+Morse-1996-375-341	341	podrido	rotten		1728	ROTTEN
+Morse-1996-375-342	342	liso	smooth		1234	SMOOTH
+Morse-1996-375-343	343	recto	straight		1404	STRAIGHT
+Morse-1996-375-344	344	correcto	correct		1725	CORRECT (RIGHT)
+Morse-1996-375-345	345	izquierdo	left		2182	LEFT HAND
```

Comment:
This could be mapped to 244 LEFT.

Reply:
Yes, I definitely deleted the wrong mappping here. Thank you!